python/jacobiandebug/analyzejacobian.py

```diff
@@ -397,10 +396,21 @@ def parseOutput(output, dofdata) :
     try:
         child = subprocess.Popen(mooseparams, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
         data = child.communicate()[0].decode("utf-8")
-        child.wait()
+        # analyze return code
+        if child.returncode == -11 :
+            print("The moose application crashed with a segmentation fault (try recompiling)")
+            sys.exit(1)
+        elif child.returncode != 1 :
```

Comment:
Won't this include cases where ``returncode == 0``?

Reply:
Yes, it better. We expect a return code of 1 if the PETSc finite differencing worked.

Reply:
Ah OK, That's good to know.